enclave/core/sgx/report.c

```diff
@@ -393,36 +358,24 @@ void oe_free_report(uint8_t* report_buffer)
     oe_free(report_buffer);
 }
 
-oe_result_t _handle_get_sgx_report(uint64_t arg_in)
+uint32_t oe_get_sgx_report_ecall(
```

Comment:
Not clear why this can't return oe_result_t

Reply:
I avoided the use of oe_result_t in EDL because I am not completely confident we can make size guarantees on all future platforms between host and enclave.

Reply:
After thinking more about this I switched to using **oe_result_** in the EDL. If there is a problem, at least it will not be with internal APIs.